documentation/modules/configuring/con-skipping-scale-down-checks.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// con-scaling-kafka-clusters.adoc
+
+[id='con-skipping-scale-down-checks-{context}']
+= Skipping checks on scale-down operations
+
+[role="_abstract"]
+By default, Strimzi performs a check to ensure that there are no partition replicas on a broker before initiating a scale-down operation on a Kafka cluster. 
+If replicas are found, the scale-down operation is blocked to prevent potential data loss. 
+To fix this, no replicas must be left on the broker before trying to scale it down again.
+
+However, there may be scenarios where you want to bypass this blocking mechanism.
+Disabling the check might be necessary on busy clusters, for example, because new topics keep generating replicas for the broker.
+This situation can indefinitely block the scaling down of clusters, even when brokers are nearly empty. 
```

Comment:
As of now, it will block the reconciliation -> not just the scale down. I.e. the cluster will not be operated. Should we make it more clear?

Reply:
Changed to: _This situation can indefinitely block cluster operations, even when brokers are nearly empty._

and also the intro:

_If replicas are found, cluster operations are blocked to prevent potential data loss. 
To resume cluster operations, no replicas must be left on the broker before trying to scale it down again._